Source/WebCore/animation/WebAnimation.cpp

```diff
@@ -1611,6 +1611,11 @@ bool WebAnimation::computeRelevance()
     if (m_playbackRate < 0 && timing.phase == AnimationEffectPhase::After)
         return true;
 
+    // - the animation effect is associated with an animation not in the idle play state with a non-null
+    // associated timeline that is not monotonically increasing.
+    if (m_timeline && !m_timeline->isMonotonic() && playState() != PlayState::Idle)
```

Comment:
Can we put the playState check first so the sentence reads exactly like the code?

Reply:
We _could_, but it's less work to do it that way since calling `playState()` has some more involved logic.